src/site/stages/build/plugins/create-react-pages.js

```diff
@@ -29,6 +29,14 @@ function createTemporaryReactPages(options) {
               path: trimmedUrl,
               contents: new Buffer(`
               <nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs" id="va-breadcrumbs">
+                <div class="vads-l-grid-container">
```

Comment:
Maybe use `vads-l-grid-container--full` so that it lines up better with the edges of the other elements.